ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -470,8 +486,15 @@ void ActiveProvider_Disconnect(VirtualizationRootHandle rootIndex)
 
         VirtualizationRoot* root = &s_virtualizationRoots[rootIndex];
         assert(nullptr != root->providerUserClient);
+        assertf(userClient == root->providerUserClient, "ActiveProvider_Disconnect: disconnecting provider IOUC %p for root index %d (%s), but expecting IOUC %p",
```

Comment:
Do the messages we print in `assertf` show up in kernel panic reports?

Reply:
Yes!

Reply:
Great thanks! 😄